### ISSUE TITLE
Stacktrace in Log

### ISSUE THREAD

**Issue by svemeyer** [dCache/dcache#5340]
Hey,

I see following stack trace in dCacheDomain.log:
```
03 Mar 2020 11:23:39 (Frontend-xxxxx) [] Uncaught exception in thread pool-56-thread-1
java.lang.NullPointerException: null
        at org.dcache.util.collector.pools.PoolInfoCollectorUtils.mergeLastAccess(PoolInfoCollectorUtils.java:326)
        at org.dcache.restful.util.pool.PoolHistoriesHandler.getAggregateFileLifetime(PoolHistoriesHandler.java:193)
        at org.dcache.restful.util.pool.PoolHistoriesHandler.update(PoolHistoriesHandler.java:265)
        at org.dcache.util.collector.pools.PoolInfoAggregator.lambda$aggregateDataForPoolGroups$0(PoolInfoAggregator.java:100)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1675)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
        at org.dcache.util.collector.pools.PoolInfoAggregator.aggregateDataForPoolGroups(PoolInfoAggregator.java:95)
        at org.dcache.restful.util.pool.PoolDataRequestProcessor.postProcess(PoolDataRequestProcessor.java:105)
        at org.dcache.util.collector.RequestFutureProcessor.waitUntilFinished(RequestFutureProcessor.java:233)
        at org.dcache.util.BoundedExecutor$Worker.run(BoundedExecutor.java:251)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
```
It occured more than one time and I'm not sure if the stacktrace should be in the logs.

**Comment by alrossi** [comment]
The NPE problem has cropped up in a number of situations.  

Similar to https://github.com/dCache/dcache/issues/4313

And I also think another one more recently ...

This looks like another path.  I'll have a look

**Comment by alrossi** [comment]
https://rb.dcache.org/r/12247/

**Comment by femiadeyemi** [comment]
Fix committed: master@bf147085e30de136f69a399c92d65c54da662452
